rm/rm-client/src/main/java/org/ow2/proactive/resourcemanager/core/properties/PAResourceManagerProperties.java

```diff
@@ -180,6 +181,19 @@
     /** Frequency of node history removal (cron expression) */
     RM_HISTORY_REMOVAL_CRONPERIOD("pa.rm.history.removal.cronperiod", PropertyType.STRING, "*/10 * * * *"),
 
+    /**
+     * Defines the frequency of attempts to remove {@link NodeState#DOWN} or
+     * {@link NodeState#LOST} node (cron expression)
+     */
+    RM_UNAVAILABLE_NODES_REMOVAL_FREQUENCY("pa.rm.nodes.unavailable.removal.frequency", PropertyType.STRING, "*/30 * * * *"),
```

Comment:
you can remove the default value here also. When it is not set, no cron scheduler will be started.

Reply:
Ok, this is done.